### PR TITLE
Fixture event id logger

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_alerter/alerter.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def merged_tags(self, check, trigger):
     def send_alert(self, check: dict, trigger: dict, event_type: str):
         self.statsd.incr('send_alert', 1)
         with self.statsd.timer('send_alert'):
-            logger.info('Sending alert `{}` for check id=`{}`'.format(
+            logger.info('Sending alert {} for check id={}'.format(
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: this should rather be
```python
            logger.info('Sending alert {} for check id={}',
                  event_type, check['id']
            )
```
(notice the lack of .format() )

File: alamo_alerter/alerter.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ def send_alert(self, check: dict, trigger: dict, event_type: str):
                     'check_name': check.get('name'),
                     'entity_name': check.get('entity_name'),
                     'trigger_name': trigger.get('name'),
-                    'eventid': '{}'.format('event_id'),
+                    'eventid': trigger.get('id'),
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: sure?

Comment (ch40s3k) [inline alamo_alerter/alerter.py]: It was under big consideration, and we decided that it gonna fit for now

Comment (deejay1) [inline alamo_alerter/alerter.py]: :ok_woman: Just fix the loggers when you're at it ;)

**Comment by pnuckowski** [comment]
:+1:

**Comment by h00dy** [comment]
:+1:

**Pull request merged**